expo/ticketing/views.py

```diff
@@ -130,8 +131,9 @@ def ticket_item_edit(request, item_id=None):
                                 urlconf='ticketing.urls',
                                 args=[str(item.bpt_event.conference.conference_slug)]))
             else:
-                error = 'ERROR:  Cannot remove Ticket Item:  \
-                        It is used in a Transaction.'
+                error = 'Cannot remove Ticket Item: %s \
```

Comment:
This change to the literal string causes a test to break:

```  
File "/vagrant/expo/tests/ticketing/test_edit_ticket.py", line 176, in test_delete_ticket_with_transactions
    nt.assert_true('ERROR' in response.content)
AssertionError: False is not true
```

Please resolve this - get in touch if you're not sure how to do this.